meta.yml

```diff
@@ -0,0 +1,116 @@
+---
+fullname: Zorn's Lemma
+shortname: zorns-lemma
+organization: coq-community
+
+synopsis: This library develops some basic set theory.
+
+description: |
+  This library develops some basic set theory.
+  The main purpose the author had in writing it was as support for the Topology library.
+
+authors:
+- name: Daniel Schepler <dschepler@gmail.com>
```

Comment:
I am unsure we should leave the e-mail of the original author in the README...

Reply:
I've included the e-mail because it was in the description and there is no contact information in the copyright notice, yet I didn't want to touch it anyhow. But sure technically there in the history is all the information anyway.

Reply:
Yep, I'm not really sure what is the right move here... :confused:

Reply:
Well, we could just CC him, I guess [this](https://github.com/dschepler) is the account.

Reply:
Yes, indeed. Let's just ask @dschepler.

Reply:
How about we just keep an optional `email` field alongside `name` for authors in `meta.yml`? Then we could use that field when available, in, for example, an `opam` file.

Reply:
Sounds even better than the <> notation. But I'd prefer `e-mail`, with the hyphen, if the tag is to be created anyway.